### PR TITLE
Update SV-COMP perf test tooling

### PR DESCRIPTION
**Pull request by tautschnig**
SV-COMP's infrastructure moved to gitlab.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: scripts/perf-test/build.yaml [hunk 1]
```diff
@@ -106,7 +106,7 @@ Resources:
                     git log HEAD^..HEAD >> COMMIT_INFO
                     make -C src minisat2-download glucose-download cadical-download
                     export CCACHE_NOHASHDIR=1
-                    make -C src CXX="ccache g++" -j8
+                    make -C src cbmc.dir goto-cc.dir CXX="ccache g++" -j8
```
Comment (TGWDB) [inline scripts/perf-test/build.yaml:109]: Are these directories properly defined?

File: scripts/perf-test/build.yaml [hunk 2]
```diff
@@ -168,9 +168,9 @@ Resources:
                     export CCACHE_NOHASHDIR=1
                     if [ x${UsePerf} = xTrue ]
                     then
-                      make -C src CXX="ccache g++" -j8 CXXFLAGS="-O2 -g -Wno-deprecated -fno-omit-frame-pointer"
+                      make -C src cbmc.dir goto-cc.dir CXX="ccache g++" -j8 CXXFLAGS="-O2 -g -Wno-deprecated -fno-omit-frame-pointer"
```
Comment (TGWDB) [inline scripts/perf-test/build.yaml:171]: Again, properly defined?

File: scripts/perf-test/build.yaml [hunk 3]
```diff
@@ -168,9 +168,9 @@ Resources:
...
                     else
-                      make -C src CXX="ccache g++" -j8 CXXFLAGS="-O2 -pg -g -finline-limit=4 -Wno-deprecated" LINKFLAGS="-pg"
+                      make -C src cbmc.dir goto-cc.dir CXX="ccache g++" -j8 CXXFLAGS="-O2 -pg -g -finline-limit=4 -Wno-deprecated" LINKFLAGS="-pg"
```
Comment (TGWDB) [inline scripts/perf-test/build.yaml:173]: Again, properly defined?

File: scripts/perf-test/ebs.yaml [hunk 4]
```diff
@@ -43,8 +46,8 @@ Resources:
                 apt-get -y update
                 apt-get install git
                 cd /mnt
-                git clone --depth 1 --branch svcomp20 \
-                        https://github.com/sosy-lab/sv-benchmarks.git
+                git clone --depth 1 --branch main \
```
Comment (TGWDB) [inline scripts/perf-test/ebs.yaml:49]: Main may not be stable, is this intended (there are other branches that might be stable)?

File: scripts/perf-test/run-on-ec2.sh [hunk 5]
```diff
@@ -113,12 +113,12 @@ do
         perf-test-sqs-:PERFTESTID:
       aws --region :AWSREGION: cloudformation delete-stack --stack-name \
         perf-test-exec-:PERFTESTID:
-      poweroff
     fi
 
     # the queue is gone, or other host will be turning
     # off the lights
     poweroff
+    sleep 10
```
Comment (TGWDB) [inline scripts/perf-test/run-on-ec2.sh:121]: Why sleep?

**Comment by tautschnig** [comment]
Closing in favour of #7003.

**Pull request closed without merging**